cypress/support/index.js

```diff
@@ -15,7 +15,27 @@
 
 // Import commands.js using ES2015 syntax:
 require('./commands');
-require('cypress-axe');
+
+const { A11yReporter } = require ('@cdssnc/a11y-tracker-client');
+
+// default to not reporting
+A11yReporter.configure({
+  trackerURI: undefined,
+  revision: '<local>',
+  project: 'cppd-medical-report-sails',
+});
+
+// if we're in CI and on the master branch, do the actual reporting
+if (Cypress.env['testing'] && Cypress.env['GITHUB_REF'] === 'refs/heads/master') {
```

Comment:
@maxneuvians QQ where is the Cypress.env[' '] configurations coming from? I don't seem to see any added to cypress.json

Reply:
You are right - I thought they would be populated automatically from `env` but they should just be set manually according to https://docs.cypress.io/api/cypress-api/env.html#Syntax

so we are missing:

```
Cypress.env('testing', true)
Cypress.env('GITHUB_REF', process.env. GITHUB_REF)
```

or it could be refactored to:

```
if (process.env.NODE_ENV === 'testing' && process.env.GITHUB_REF === 'refs/heads/master')
```

Reply:
The cypress environment variables look like they are just for testing environment variable changes  so I think this should just use the process.env I'll make the change.